kong/meta.lua

```diff
@@ -20,6 +20,6 @@ return {
   -- third-party dependencies' required version, as they would be specified
   -- to lua-version's `set()` in the form {from, to}
   _DEPENDENCIES = {
-    nginx = { "1.19.3.1", "1.19.9.1" },
+    nginx = { "1.21.4.1" },
```

Comment:
not a blocker, but i'm curious on the reason behind this restriction, is there a specific behaviour we rely on that doesn't work with previous versions of openresty?

Reply:
@fffonion It is quite major change in Nginx, the behavior is very different, as seen in the changed tests. Also sooner rather than later we start to utilize things like `string` buffer with LuaJIT that is only available in latest.

There is nothing in particular that even lower versions couldn't be supported, it is just that nobody these days can just use plain OpenResty, you always need to include our patches etc. So why would anyone go even the hassle and build it with old OpenResty if they need to build it.